adapters/beintoo/beintoo.go

```diff
@@ -0,0 +1,256 @@
+package beintoo
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type BeintooAdapter struct {
+	endpoint string
+	testing  bool
+}
+
+func buildEndpoint(endpoint string, testing bool, timeout int64) string {
+	
+	if timeout == 0 {
+		timeout = 1000
+	}
+	
+	uriObj, _ := url.Parse(endpoint)
+	parameters := url.Values{}
+	
+	if testing {
+		
+		parameters.Add("t", "1000")
+		parameters.Add("ts", "2060541160")
+		uriObj.RawQuery = parameters.Encode()
+
+	return uriObj.String()
+	}
+	
+	parameters.Add("t", strconv.FormatInt(timeout, 10))
+	parameters.Add("ts", strconv.FormatInt(time.Now().Unix(), 10))
+	parameters.Add("src", "pbserver")
+
+	uriObj.RawQuery = parameters.Encode()
+
+	return uriObj.String()
```

Comment:
Looks nice. Can we add a test case were `testing` is set to false so lines 40 to 46 get covered?
![image](https://user-images.githubusercontent.com/15863840/81080573-df545d80-8ebe-11ea-8716-8297965d7676.png)

Reply:
yes